bazel/cython_library.bzl

```diff
@@ -46,7 +46,7 @@ def pyx_library(name, deps=[], py_deps=[], srcs=[], **kwargs):
             # Optionally use PYTHON_BIN_PATH on Linux platforms so that python 3
             # works. Windows has issues with cython_binary so skip PYTHON_BIN_PATH.
             cmd=
-            "PYTHONHASHSEED=0 $(location @cython//:cython_binary) --cplus $(SRCS) --output-file $(OUTS)",
+            "PYTHONHASHSEED=0 $${PYTHON_BIN_PATH} $(location @cython//:cython_binary) --cplus $(SRCS) --output-file $(OUTS)",
```

Comment:
This file was copied from grpc, right? Was there a bug in the grpc version or are our requirements somehow different?

Reply:
I think the GRPC version will also run into the same problems, I'm quite confident that this is the right fix (because it even works without any python on the path, so we can be somewhat sure it won't pick up a different python version).